src/WatchDogs.FakeSource/TradeModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Bogus;
+
+namespace WatchDogs.FakeSource;
+
+public record TradeModel
```

Comment:
Tim likes to add Model suffix to the classes names which makes sense in context of MVC apps, but I would omit it here.

Reply:
Done.